frontend/src/bundles/auth/components/auth-layout/styles.module.scss

```diff
@@ -146,3 +146,96 @@
         height: 50px;
     }
 }
+
+// FORMS CSS
+:global(.form) {
```

Comment:
can we add some class to form and use global only for this class? 

```scss
:global(.formWrapper) {
     // your styles
}
```

Reply:
I have been thinking about this a long time, and I'm not sure if this is possible. Because if only .form is global I would have to transfer styles from for `.label`, `.footer`, `.cta` to the sign-in sign-up styles modules respectively.

This would result in a duplication between sign-in and sign-up style modules.

Reply:
It works now